### PR TITLE
br: update tiflash related docs on nfs

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a notice about TiFlash on BR user case.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@3pointer Please involve a technical review.

**Comment by 3pointer** [comment]
@glorv PTAL

File: br/backup-and-restore-use-cases.md [hunk 1]
```diff
@@ -100,9 +100,14 @@ BR 工具已支持自适应 GC，会自动将 `backupTS`（默认是最新的 PD
...
 
 * [备份前的准备工作](#备份前的准备工作)。
-* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
+* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点 (TiFlash 节点) 为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
 * NFS server 和 NFS client 间的数据传输速率至少要达到备份集群的 `TiKV 实例数 * 150MB/s`。否则网络 I/O 有可能成为性能瓶颈。
 
+> **注意：**
+>
+> * 因为备份时候只备份单副本 (leader) 数据，所以即使集群中存在 TiFlash 副本，无需挂载 TiFlash 节点也能完成备份。
+> * 但是恢复的时候需要恢复全部副本的数据，因此在恢复时 TiFlash 节点需要有备份数据的访问权限才能完成恢复，所以建议此时也一并将 TiFlash 节点挂载到 NFS server 上 。
```
Comment (glorv) [inline br/backup-and-restore-use-cases.md:109]:
```suggestion
> * 但是恢复的时候需要恢复全部副本的数据，因此在恢复时 TiFlash 节点需要有备份数据的访问权限才能完成恢复，所以此时也必须将 TiFlash 节点挂载到 NFS server 上 。
```
这里语气不应该是建议这么弱，如果包含需要恢复的 tiflash 副本， 则 tiflash 必须挂 nfs, 否则恢复必定会失败

File: br/backup-and-restore-use-cases.md [hunk 2]
```diff
@@ -100,9 +100,14 @@ BR 工具已支持自适应 GC，会自动将 `backupTS`（默认是最新的 PD
 #### 前置要求
 
 * [备份前的准备工作](#备份前的准备工作)。
-* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
+* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点 (TiFlash 节点) 为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
```
Comment (glorv) [inline br/backup-and-restore-use-cases.md:103]:
```suggestion
* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点、 TiKV 节点 和 TiFlash 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
```

**Comment by TomShawn** [comment]
@glorv PTAL again, thanks~

**Comment by glorv** [comment]
LGTM

File: br/backup-and-restore-use-cases.md [hunk 3]
```diff
@@ -100,9 +100,14 @@ BR 工具已支持自适应 GC，会自动将 `backupTS`（默认是最新的 PD
...
 
 * [备份前的准备工作](#备份前的准备工作)。
-* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
+* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点、 TiKV 节点 和 TiFlash 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
 * NFS server 和 NFS client 间的数据传输速率至少要达到备份集群的 `TiKV 实例数 * 150MB/s`。否则网络 I/O 有可能成为性能瓶颈。
 
+> **注意：**
+>
+> * 因为备份时候只备份单副本 (leader) 数据，所以即使集群中存在 TiFlash 副本，无需挂载 TiFlash 节点也能完成备份。
+> * 但是恢复的时候需要恢复全部副本的数据，因此在恢复时 TiFlash 节点需要有备份数据的访问权限才能完成恢复，所以此时也必须将 TiFlash 节点挂载到 NFS server 上 。
```
Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:109]:
```suggestion
> * 恢复时需要恢复全部副本的数据，因此在恢复时 TiFlash 节点需要有备份数据的访问权限才能完成恢复，所以此时必须将 TiFlash 节点挂载到 NFS server 上 。
```

Comment (TomShawn) [reply br/backup-and-restore-use-cases.md:109]: @3pointer PTAL

Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:109]:
```suggestion
> * BR 在恢复数据时，会恢复全部副本的数据。因此在恢复时，TiFlash 节点需要有备份数据的访问权限 TiFlash 才能完成恢复，此时也必须将 TiFlash 节点挂载到 NFS server 上 。
```

File: br/backup-and-restore-use-cases.md [hunk 4]
```diff
@@ -100,9 +100,14 @@ BR 工具已支持自适应 GC，会自动将 `backupTS`（默认是最新的 PD
 #### 前置要求
 
 * [备份前的准备工作](#备份前的准备工作)。
-* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点和 TiKV 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
+* 配置一台高性能 SSD 硬盘主机为 NFS server 存储数据。其他所有 BR 节点、 TiKV 节点 和 TiFlash 节点为 NFS client，挂载相同的路径（例如 `/br_data`）到 NFS server 上以访问 NFS server。
 * NFS server 和 NFS client 间的数据传输速率至少要达到备份集群的 `TiKV 实例数 * 150MB/s`。否则网络 I/O 有可能成为性能瓶颈。
 
+> **注意：**
+>
+> * 因为备份时候只备份单副本 (leader) 数据，所以即使集群中存在 TiFlash 副本，无需挂载 TiFlash 节点也能完成备份。
```
Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:108]: “也能完成备份”的主体是 TiFlash 还是 BR？
```suggestion
> * 因为备份时候只备份单副本 (leader) 数据，所以即使集群中存在 TiFlash 副本，无需挂载 TiFlash 节点也能完成备份。
```

**Pull request merged**